libs/feature/search/src/lib/utils/service/fields.ts

```diff
@@ -220,13 +234,78 @@ if (doc.containsKey('licenseObject.default.keyword') && doc['licenseObject.defau
 if (doc.containsKey('MD_LegalConstraintsUseLimitationObject.default.keyword') && doc['MD_LegalConstraintsUseLimitationObject.default.keyword'].length > 0)
   raw += doc['MD_LegalConstraintsUseLimitationObject.default.keyword'].join('|').toLowerCase();
 
-if (raw.contains('odbl')) emit('ODbL');
-if (raw.contains('pddl')) emit('PDDL');
-if (raw.contains('odc-by')) emit('ODC-By');
-if (raw.contains('cc0') || raw.contains('cc-0')) emit('CC-0');
-if (raw.contains('cc-by') || raw.contains('cc by')) emit('CC BY');
-if (raw.contains('open license')) emit('Open License');
-if (raw.contains('etalab')) emit('Etalab');`
+boolean unknown = true;
```

Comment:
This file is going to be huge in the long run.
Might be easier to decouple each field in a dedicated file, wdyt ?

Reply:
Definitely, one file per class of field could make sense; I'm going to refactor things a bit to take less space